v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-model.ts

```diff
@@ -48,12 +49,16 @@ export const UnivariateMeasureAdornmentModel = AdornmentModel
     },
     get isUnivariateMeasure() {
       return true
+    },
+    computeMeasureRange(attrId: string, cellKey: Record<string, string>, dataConfig: IDataConfigurationModel) {
+      // derived models should override if they have a range
+      return undefined
+    },
+    computeMeasureValue(attrId: string, cellKey: Record<string, string>, dataConfig: IDataConfigurationModel) {
```

Comment:
Note that I made this a view instead of an action. Even in the individual "sub adornment" models, `computeMeasureValue` doesn't set or change any model properties, and is already defined as a view, not an action.